setup.py

```diff
@@ -5,9 +5,11 @@
 
 from setuptools import setup
 
+with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as fp:
+  LONG_DESCRIPTION = fp.read() + '\n'
```

Comment:
here's a preview of what this concatenation would look like: https://gist.github.com/kwlzn/97f740834edcc03b6e729a3b18d641dc

note the disjoint styling between the concatenation. seems to me that we'd probably want to change the style of the CHANGES.rst header, which is currently:

```
=======
CHANGES
=======
```

to something like:

```
Release Notes
===========
```

to make this line up nicely.